node/full.go

```diff
@@ -27,7 +28,7 @@ import (
 	"github.com/rollkit/rollkit/pkg/service"
 	"github.com/rollkit/rollkit/pkg/signer"
 	"github.com/rollkit/rollkit/pkg/store"
-	"github.com/rollkit/rollkit/pkg/sync"
+	rollkitSync "github.com/rollkit/rollkit/pkg/sync"
```

Comment:
tiny nit: package name full lowercase, i thought it was a variable when reading the code on gh because of this.